tincr/cad/design/cells.tcl

```diff
@@ -468,3 +469,22 @@ proc ::tincr::reset_configuration {cell config_list} {
         set_property $config $default $cell  
     }
 }
+
+## Creates and returns a list of properties on the specified cell object that are configurable.
+#   TODO: cache this information
+#
+# @param cell Vivado cell instance
+# @return A set of reconfigurable properties on that cell  
+#
+proc tincr::get_configurable_properties {cell} {
```

Comment:
Correct declaration for ensemble proc:
`proc tincr::cells::get_configurable_properties { cell }`